progress/progress.go

```diff
@@ -43,7 +43,7 @@ func NewModel(size int) *Model {
 	return &Model{
 		StartColor:     startColor,
 		EndColor:       endColor,
-		Length:         size,
+		Width:         size,
```

Comment:
Please make sure your code is formatted with `gofmt`. I can recommend enabling that as a save-hook in your editor/IDE.

Reply:
I edited via github ide, sorry, i didn't note that

Reply:
Don't worry, it happens :smiley: